members.md

```diff
@@ -171,6 +211,32 @@ Status: 200 OK
 
 ***
 
+## Ban Member
```

Comment:
We need to make sure it's very clear that this endpoint is v2, it's very easy for someone to miss the note 


```suggestion
## Ban Member (v2)
```